resource/closecaption_german.txt

```diff
@@ -800,45 +800,45 @@
 "[english]crash.NoAmmo0"		"<clr:238,180,34>Crash: Out of ammo!"
 "crash.Medic0"		"<clr:238,180,34>Crash: Sanitäter!"
 "[english]crash.Medic0"		"<clr:238,180,34>Crash: Medic!"
-"crash.Reloading0"		"<clr:238,180,34>Crash: Sichern und laden!"
+"crash.Reloading0"		"<clr:238,180,34>Crash: Durchladen!"
 "[english]crash.Reloading0"		"<clr:238,180,34>Crash: Lock and load!"
-"crash.Reloading1"		"<clr:238,180,34>Crash: Nachladen, gib mir Deckung!"
+"crash.Reloading1"		"<clr:238,180,34>Crash: Nachladen, gebt mir Deckung!"
 "[english]crash.Reloading1"		"<clr:238,180,34>Crash: Reload, cover me!"
 "crash.Reloading2"		"<clr:238,180,34>Crash: Nachladen!"
 "[english]crash.Reloading2"		"<clr:238,180,34>Crash: Reload!"
 "crash.Reloading3"		"<clr:238,180,34>Crash: Nachladen!"
 "[english]crash.Reloading3"		"<clr:238,180,34>Crash: Reload!"
-"crash.SentryInPlace0"		"<clr:238,180,34>Crash: Sentry Gun aktiv!"
+"crash.SentryInPlace0"		"<clr:238,180,34>Crash: Geschützturm aktiviert!"
 "[english]crash.SentryInPlace0"		"<clr:238,180,34>Crash: Sentry gun active!"
-"crash.SentryInPlace1"		"<clr:238,180,34>Crash: Sentry Gun aktiv!"
+"crash.SentryInPlace1"		"<clr:238,180,34>Crash: Geschützturm aktiviert!"
 "[english]crash.SentryInPlace1"		"<clr:238,180,34>Crash: Sentry gun active!"
-"crash.SentryInPlace2"		"<clr:238,180,34>Crash: Die Wache ist in Position."
+"crash.SentryInPlace2"		"<clr:238,180,34>Crash: Geschützturm in Position."
 "[english]crash.SentryInPlace2"		"<clr:238,180,34>Crash: Sentry's in place."
-"crash.SentryInPlace3"		"<clr:238,180,34>Crash: Wache aufgebaut."
+"crash.SentryInPlace3"		"<clr:238,180,34>Crash: Geschützturm montiert."
 "[english]crash.SentryInPlace3"		"<clr:238,180,34>Crash: Sentry assembled."
-"crash.SentryInPlace4"		"<clr:238,180,34>Crash: Sentry online."
+"crash.SentryInPlace4"		"<clr:238,180,34>Crash: Geschützturm aktiviert."
 "[english]crash.SentryInPlace4"		"<clr:238,180,34>Crash: Sentry online."
-"crash.SentryInPlace5"		"<clr:238,180,34>Crash: Die Wache ist bereit, ein paar Käfer zu töten."
+"crash.SentryInPlace5"		"<clr:238,180,34>Crash: Geschützturm einsatzbereit. Machen wir sie platt!"
 "[english]crash.SentryInPlace5"		"<clr:238,180,34>Crash: Sentry's ready to waste some bugs."
 "crash.Ammo0"		"<clr:238,180,34>Crash: Munition."
 "[english]crash.Ammo0"		"<clr:238,180,34>Crash: Ammo."
 "crash.Medkit0"		"<clr:238,180,34>Crash: Erste-Hilfe-Spritze."
 "[english]crash.Medkit0"		"<clr:238,180,34>Crash: Medkit."
 "crash.Thanks0"		"<clr:238,180,34>Crash: Danke."
 "[english]crash.Thanks0"		"<clr:238,180,34>Crash: Thanks."
-"crash.Thanks1"		"<clr:238,180,34>Crash: Danke!"
+"crash.Thanks1"		"<clr:238,180,34>Crash: Danke schön."
 "[english]crash.Thanks1"		"<clr:238,180,34>Crash: Thank you."
 "crash.Question0"		"<clr:238,180,34>Crash:"
 "[english]crash.Question0"		"<clr:238,180,34>Crash:"
```

Comment:
@BenLubar: Should the original line of `crash.Question0` maybe `Crash: [Silence]` or `Crash: ???`?